### PR TITLE
Fetch events from FB periodically

### PR DESCRIPTION
**Pull request by rht**
Tested the fetching and db insertion. Note that:
- [x] the entry is not dedup-ed so it needs a unique key as for the event id (could use from the existing fb id or hashed from the title).
- [x] if the event url could be added
- [x] the timestamp has yet to be made UNIXTIME

### CODE REVIEW

**Comment by rht** [comment]
Done! Now this needs a human `$ROOT` to manually altertable to add the url and id column as seen in the schema here https://github.com/fxfactorial/silicondzor/pull/21/commits/8f205d9feaad6c46ce4ece4fc40d62330d9e52f0 (tho silicondzor{bot} could do this as well, if being told how to do so through a script).

**Comment by fxfactorial** [comment]
This is AWESOME! Great job! and so so appreciated! I'll make the relevant changes and deploy soon. So awesome!

File: backend/server.js [hunk 1]
```diff
@@ -54,9 +55,37 @@ const Routes = require('../lib/routes').default;
...
+          $end: each.end_time ? Math.floor(Date.parse(each.end_time)/1000) : 0,
+          $description: each.description,
+          $creator: group_name,
+          $url: `https://facebook.com/events/${each.id}`,
+          $id: `fb-${each.id}`
+        });
+      });
+    });
+  }
+}, 60 * 1000 * 60 * 48);
```
Comment (rht) [inline backend/server.js:87]: this runs only 48hrs after

**Comment by fxfactorial** [comment]
For people wondering how to change the table, I did: 

```
alter table event add column url text not null default '';
```

**Pull request merged**